addons/website_forum/static/src/js/website_forum.js

```diff
@@ -27,6 +27,7 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         'click .o_wforum_favourite_toggle': '_onFavoriteQuestionClick',
         'click .comment_delete': '_onDeleteCommentClick',
         'click .js_close_intro': '_onCloseIntroClick',
+        'click .o_wforum_submit_post': '_onSubmitForm'
```

Comment:
- Missing trailing comma.
- Why not on submit?

Reply:
How does it will interact with the submit listener in website_forum.share.js ?
`'submit .js_website_submit_form:has(:not(.karma_required).oe_social_share_call)': '_onShareCallClick',`
Afraid it will trigger that event even if the title or the textarea is empty

Reply:
Ideally, both widgets should listen to submit, the first one should preventDefault the event if the form is not correct and the second one should check that the event is not default prevented. But this only works if the widgets are executed in the right order, which we cannot do. What you can do, is include the social share code directly in the same widget as the validation, is it possible?

Reply:
Yes it is. But, for now, my verification is in a website forum widget that inherit from public widget. So, this verification is only available in website_forum. But if we want it available for every website module, it should be redirected to website. And so, the social share code will no longer be in the same widget because it's a website_forum feature.
I guess it can stick with it even in website because it will only be triggered by submit buttons with a specific class (an these buttons will only be in website_forum views). But it will feel wrong to have a website_forum feature in website.
So, I guess it depends if we want the verification being available for the whole website or only for website_forum

Reply:
It would still be possible with an `include`. Anyways, do it just in forum for now.